ocpp/routing.py

```diff
@@ -1,7 +1,7 @@
 import functools
 
 
-def on(action):
+def on(action, *, skip_schema_validation=False):
```

Comment:
Where does the asterisk come into play?

Reply:
All arguments after the aterisk can only used with as keyword arguments.

So the following call would raise a TypeError:

``` python
def foo(*, a):
    pass

foo(1)  # will raise a TypeError
```

Instead it is required to use keyword arguments:

``` python
foo(a=1)  # works fine
```

 Without aterisk the `@on()` decorator could be used like this:

``` python
@on("MeterValues", True)
def on_meter_values(*args, **kwargs):
   pass
```

Without the keyword it is hard to guess what the meaning is of the second argument. I added the aterisk to make it explicit to disable validation. You can either use `skip_schema_validation`s default value of `False`. Or, if you want to disable the validation, you can set it to true by keyword arguments:

``` python
@on("MeterValues", skip_schema_validation=True)
def on_meter_values(*args, **kwargs):
   pass
```

Reply:
Interesting. Thanks for the explanation.